docs/source/contributing/developing-core.md

```diff
@@ -252,6 +252,14 @@ pnpm start
 
 You can also run commands of specific workspaces using the `--filter` feature as shown in the previous section.
 
+```{note}
```

Comment:
@stevepiercy So I felt the need to add this info after looking at 
https://6.docs.plone.org/volto/contributing/developing-core.html#developing-volto

I will revert this change if you think that it's pointless because we already said something
similar here https://6.docs.plone.org/volto/contributing/index.html#code-quality